test/message/promise_fast_reject.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
This makes sure that we don't log the first or last error in this case which get handled. I'm not really sure what you are asking but it makes sense if you read the impl for the "throw" on regular exit case.